flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/AggregateFunction.scala

```diff
@@ -35,6 +36,18 @@ abstract class AggregateFunction[T] extends UserDefinedFunction {
   def createAccumulator(): Accumulator
 
   /**
+    * Retract the input values from the accumulator instance.
+    *
+    * @param accumulator the accumulator which contains the current
+    *                    aggregated results
+    * @param input       the input value (usually obtained from a new arrived data)
```

Comment:
Ah, OK. but I think the above comments should be clear enough: "Retract the input values from the accumulator instance" I think we can leave it as this because it makes accumulate and retract interface consistent and clean. We will have (user defined) multiple inputs for both accumulate and retract in the near future, at which point we will anyway not control the naming of this.